mimesis/providers/numbers.py

```diff
@@ -66,6 +66,32 @@ def complexes(self, start_real: float = 0, end_real: float = 1,
                     self.random.uniform(start_imag, end_imag, rounding_imag))
             for _ in range(n)]
 
+    def matrix(self, m: int = 10, n: int = 10, num_type: str = 'floats',
```

Comment:
It's better to create enum object:

```
def matrix(self, m: int = 10, n: int = 10, num_type: str = NumType.FLOAT)
```

I think that the idea is pretty clear.